### PR TITLE
Fix some merge errors

### PR DESCRIPTION
**Pull request by erikzhang**

### CODE REVIEW

**Review by shargon — DISMISSED** [review]

File: src/neo/Network/P2P/Payloads/MerkleBlockPayload.cs [hunk 1]
```diff
@@ -34,7 +34,7 @@ public static MerkleBlockPayload Create(Block block, BitArray flags)
         public void Deserialize(BinaryReader reader)
         {
             Header = reader.ReadSerializable<Header>();
-            TxCount = (int)reader.ReadVarInt(ProtocolSettings.Default.MaxTransactionsPerBlock);
+            TxCount = (int)reader.ReadVarInt();
```
Comment (shargon) [inline src/neo/Network/P2P/Payloads/MerkleBlockPayload.cs:37]: We should set a max

Comment (erikzhang) [reply src/neo/Network/P2P/Payloads/MerkleBlockPayload.cs:37]: Hard coded?

Comment (shargon) [reply src/neo/Network/P2P/Payloads/MerkleBlockPayload.cs:37]: Yes, `ushort.MaxValue`?

Comment (erikzhang) [reply src/neo/Network/P2P/Payloads/MerkleBlockPayload.cs]: Done.

**Review by shargon — APPROVED** [review]

**Pull request merged**